tools/ci/deps-ubuntu.sh

```
@@ -45,6 +45,18 @@ sudo apt-get -qq install -y \
   libsimde-dev
 
+# Install minizip-ng (no apt package available, build from source)
+MINIZIP_NG_VERSION=4.0.7
+wget -q "https://github.com/zlib-ng/minizip-ng/archive/refs/tags/${MINIZIP_NG_VERSION}.tar.gz" -O /tmp/minizip-ng-${MINIZIP_NG_VERSION}.tar.gz
+tar xzf /tmp/minizip-ng-${MINIZIP_NG_VERSION}.tar.gz -C /tmp
+cmake -S /tmp/minizip-ng-${MINIZIP_NG_VERSION} -B /tmp/minizip-ng-build \
+  -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON \
+  -DMZ_FETCH_LIBS=OFF -DMZ_LIB_SUFFIX=-ng -DMZ_COMPAT=OFF \
+  -DMZ_ZLIB=ON -DMZ_BZIP2=ON -DMZ_LZMA=OFF -DMZ_ZSTD=OFF \
+  -DMZ_OPENSSL=OFF -DMZ_ICONV=OFF -DMZ_PKCRYPT=ON -DMZ_WZAES=OFF
+cmake --build /tmp/minizip-ng-build -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)